### PR TITLE
[JENKINS-70102] Resizing browser window below 970px results in too narrow form elements

### PR DESCRIPTION
**Pull request by 1weiho**
See [JENKINS-70102](https://issues.jenkins.io/browse/JENKINS-70102).

Before:
截圖 2022-11-20 上午1 03 25

After:
截圖 2022-11-20 上午1 03 33

### Proposed changelog entries

- set max-width to 970px.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7419"></a>

### CODE REVIEW

**Comment by daniel-beck** [comment]
Looking at the change that introduced the code being changed here, it also added https://github.com/jenkinsci/jenkins/blob/a15914fb47cfa7274c3c430ec2a336126c28a6ae/war/src/main/less/base/layout-commons.less#L84-L86, which is not being changed here. Why is that? Is that for a different purpose? CC @janfaracik

**Comment by janfaracik** [comment]
> Looking at the change that introduced the code being changed here, it also added
> 
> https://github.com/jenkinsci/jenkins/blob/a15914fb47cfa7274c3c430ec2a336126c28a6ae/war/src/main/less/base/layout-commons.less#L84-L86
> 
> , which is not being changed here. Why is that? Is that for a different purpose? CC @janfaracik

image

The page becomes offset without that (could probably do with a refactor eventually to tidy up the vars), would be good to have alignment though - @davidho0403 would you mind updating the line @daniel-beck mentioned to be 970px too? Thanks :)

**Comment by davidho0403** [comment]
> > Looking at the change that introduced the code being changed here, it also added
> > https://github.com/jenkinsci/jenkins/blob/a15914fb47cfa7274c3c430ec2a336126c28a6ae/war/src/main/less/base/layout-commons.less#L84-L86
> > 
> > , which is not being changed here. Why is that? Is that for a different purpose? CC @janfaracik
> 
> image
> 
> The page becomes offset without that (could probably do with a refactor eventually to tidy up the vars), would be good to have alignment though - @davidho0403 would you mind updating the line @daniel-beck mentioned to be 970px too? Thanks :)

Sure~

**Review by janfaracik — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Comment by daniel-beck** [comment]
@janfaracik The other value should probably be around 1200px, otherwise it looks like this between 790 and 1200?

Screenshot 2022-12-01 at 13 39 47

**Review by ridemountainpig — APPROVED** [review]

**Comment by NotMyFault** [comment]
> @janfaracik The other value should probably be around 1200px, otherwise it looks like this between 790 and 1200?
> 
> Screenshot 2022-12-01 at 13 39 47

Gentle reminder ping 👀

File: war/src/main/less/base/layout-commons.less
Comment (janfaracik) [inline war/src/main/less/base/layout-commons.less:83]: I can't push but changing this to 1200px fixed the behaviour Daniel posted about

Comment (timja) [reply war/src/main/less/base/layout-commons.less:83]:
```suggestion
  @media screen and (max-width: 1200px) {
```

**Review by jamesD33064 — APPROVED** [review]

**Review by ridemountainpig — APPROVED** [review]

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
